### PR TITLE
Fauna frequency adjustement

### PR DESCRIPTION
**Pull request by Casulono**
# Description

The idea for limiting the amount of Fauna noises played. A player will set a level of frequency and if the random number is smaller or equal to the frequency value, then the noise will play. An additionally introduced thing is allowing to play noises in full before the next one can start. Setting frequency to max disabled this limitation

## Related issue(s)

https://github.com/Saplings-Projects/Birthday-2024/issues/102

## List of changes

A more detailed list of the changes.
- Added a new slider to the audio settings: Fauna frequency
- added "volume" to all volume settings
- Fauna noises will need to end before another one gets played
- frequency setting affects Fauna noises played with each interaction
## Additional notes

The panel holding all setting sliders needs to be enlarged. I didn't find a way to do it.

### CODE REVIEW

**Review by Mikotey — COMMENTED** [review]
Everything looks good!
Only need to adjust the Settings Window size and 1 other change

**Comment by Mikotey** [comment]
Oh, forgot to ask, does it feel better to have the fauna noises reduced? The general feel.

You made it a setting which is better so the player can decide. You think it should default to 100% or 50%?

**Comment by Casulono** [comment]
> Oh, forgot to ask, does it feel better to have the fauna noises reduced? The general feel.
> 
> You made it a setting which is better so the player can decide. You think it should default to 100% or 50%?

Personally I like the 100% and nonstop Fauna, because when you spin a piece a lot noises can't fully play and when you stop Fauna is offended by it. There's not a lot of rotate sounds so you get "What the heck?" or "Oiiiiiii"

File: Birthday-2024-Project/ScenePrefabs/SettingsWindow.tscn
[the remark this answers is not in the document]
Comment (Casulono) [reply Birthday-2024-Project/ScenePrefabs/SettingsWindow.tscn:1]: yeah, 850 does look nice

**Review by Mikotey — APPROVED** [review]

**Pull request merged**